third_party/2and3/jinja2/environment.pyi

```diff
@@ -71,6 +71,16 @@ class Environment:
     def from_string(self, source: Text, globals: Optional[Dict[str, Any]] = ..., template_class: Optional[Type[Template]] = ...) -> Template: ...
     def make_globals(self, d: Optional[Dict[str, Any]]) -> Dict[str, Any]: ...
 
+    # Frequently added extensions are included here:
+    # from InternationalizationExtension:
+    def install_gettext_translations(self, translations: Any, newstyle: Optional[bool]): ...
+    def install_null_translations(self, newstyle: Optional[bool]): ...
+    def install_gettext_callables(self, gettext: Callable, ngettext: Callable,
+                                  newstyle: Optional[bool]): ...
+    def uninstall_gettext_translations(self, translations: Any): ...
+    def extract_translations(self, source: Any, gettext_functions: Any): ...
+    newstyle_gettext = ... # type: bool
```

Comment:
flake8 is unhappy with just 1 space before the comment, see the Travis results.

On the other hand, pytype failure seems to be unrelated to these changes.